components/bl-list-component/README.md

```diff
@@ -0,0 +1,50 @@
+# List
+
+List is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows you to add a standard list to your application. Select the type of list (ordered/unordered) and specify the data that will be displayed inside the list.
+More information about list you can find [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li).
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property   | Type                                          | Default value | Logic           | Data Binding | UI Setting | Description                                                                                      |
+|------------|-----------------------------------------------|---------------|-----------------|--------------|------------|--------------------------------------------------------------------------------------------------|
+| Type       | *Select* <br> `'Unordered'` <br> `'Ordered'`  | `Unordered`   | Type Logic      | NO           | YES        | Controls the type of list(ul/ol).                                                                |
```

Comment:
There should be real values, not labels
Unordered -> ul
Ordered - ol

Reply:
Check the example
https://github.com/Backendless/ui-builder-library/blob/main/components/bl-leafletMap-component/README.md